streams/us.m3u

```diff
@@ -1905,3 +1905,53 @@ https://2-fss-2.streamhoster.com/pl_120/amlst:203530-1426406/playlist.m3u8
 https://5d80890572342.streamlock.net:1936/live/wstoLive/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",YouTV
 https://tv.sikkylab.me/hls/stream.m3u8
+#EXTINF:-1 tvg-id="K21JQD1.us" status="",Blue Mountain TV
+https://1189614805.rsc.cdn77.org/hls/bmt.m3u8
+#EXTINF:-1 tvg-id="K21JND1.us" status="",Kool TV
+http://64.91.244.153:1935/roku/roku/chunklist_w2078620659.m3u8
+#EXTINF:-1 tvg-id="WorldHarvestTV.us" status="",World Harvest TV
+https://597865f6e4114.streamlock.net/live/leseaorigin.stream/chunklist_w1838529020.m3u8
```

Comment:
```suggestion
https://597865f6e4114.streamlock.net/live/leseaorigin.stream/playlist.m3u8
```

Please replace all the chunklist_*.m3u8 links in us.m3u8 with playlist.m3u8

Thanks